requirements.txt

```diff
@@ -1,11 +1,11 @@
-dash
-dash_labs
+numpy
```

Comment:
Do you still need this file if you are managing your dependencies with Pipfiles now?

Reply:
very fair point